modules/nf-neuro/bundle/stats/main.nf

```diff
@@ -49,15 +49,29 @@ process BUNDLE_STATS {
     """
     bundles=( ${bundles.join(" ")} )
     label_map=( ${labels_map.join(" ")} )
+    metrics=( ${metrics.join(" ")} )
 
     for index in \${!bundles[@]};
     do\
         bname=\$(basename \${bundles[index]} .trk);
-        b_metrics="$metrics";
+        bname=\${bname/${prefix}__/}
+        bname=\${bname%%_labels_*}
+
+        echo "Bundle name: \${bname}"
+
+        # Initialize array for all relevant metrics
+        b_metrics=()
+
+        for m in \${!metrics[@]}; do
+            # Include if: matches bname OR is not an afd_metric file
+            if [[ "\${metrics[\$m]}" == *"\${bname}"* ]] || [[ "\${metrics[\$m]}" != *"afd_metric"* ]]; then
```

Comment:
The second condition here will associate **bundle specific metrics associated to other `bname`**. Is the `afd_metric` the only **bundle-specific** metric expected for now  ?